### PR TITLE
Dg read

### PR DESCRIPTION
**Pull request by finalelement**
Deepgrow Readme Updated for Generic App, Spleen & Left Atrium

### CODE REVIEW

File: sample-apps/generic_deepgrow/README.md [hunk 1]
```diff
@@ -2,9 +2,73 @@
 
 ### Model Overview
 
-Interactive MONAI Label App using DeepGrow (https://arxiv.org/abs/1903.08205).
+This MONAI Label App is based on DeepGrow which allows for an interactive segmentation, where 
+the user can guide the segmentation using positive and negative clicks (https://arxiv.org/abs/1903.08205).
 It uses pre-trained Deepgrow Models for NVIDIA Clara.
 
+To those extending this App, please clone the folder and adjust the following hyper-parameters:
```
Comment (diazandr3s) [inline sample-apps/generic_deepgrow/README.md:9]: Should we avoid using the word cloning for this step? There is a CLI command for this 

> monailabel apps --name generic_deepgrow --download --output myapp

File: sample-apps/generic_deepgrow/README.md [hunk 2]
```diff
@@ -2,9 +2,73 @@
...
+
+- learning rate
+  > Default value is 0.0001. Change this in the [info.YAML](./info.yaml) file
+
+- validation split used during training
+    > Default value is 0.2. Change this in the [info.YAML](./info.yaml) file
+
+### Inputs
+
+- 1 channel for the image modality -> Automated mode
```
Comment (diazandr3s) [inline sample-apps/generic_deepgrow/README.md:56]: Does deepgrow have this case? I think it always has three channels as input only, right?

File: sample-apps/generic_deepgrow/README.md (hunk 1 above)
Comment (finalelement) [reply sample-apps/generic_deepgrow/README.md:9]: let me fix this.

File: sample-apps/generic_deepgrow/README.md (hunk 2 above)
Comment (finalelement) [reply sample-apps/generic_deepgrow/README.md:56]: The user only has to deal with a single channel.

**Review by finalelement — COMMENTED** [review]
Made changes as suggested by Andres

File: sample-apps/generic_deepgrow/README.md (hunk 2 above)
Comment (diazandr3s) [reply sample-apps/generic_deepgrow/README.md:60]: By channel, I didn't mean modalities. That's why I thought deepgrow only works with three channels/tensors (image, foreground points and background points) not one. Isn't it?

**Review by aihsani — APPROVED** [review]

File: sample-apps/generic_deepgrow/README.md [hunk 3]
```diff
@@ -2,9 +2,77 @@
...
+  > - [NormalizeIntensityd](https://docs.monai.io/en/latest/_modules/monai/transforms/intensity/dictionary.html#NormalizeIntensityd) -> Normalize intensity of the image,
+
+
+  Transformations used for the clicks simulation [(DeepGrow App)](https://docs.monai.io/en/latest/apps.html)
+  > 
+  > - [FindAllValidSlicesd](https://docs.monai.io/en/latest/_modules/monai/apps/deepgrow/transforms.html#FindAllValidSlicesd) -> This transform finds/lists all valid slices in the label. Label is assumed to be a 4D Volume with shape CDHW, where C=1,
+  > - [AddInitialSeedPointd](https://docs.monai.io/en/latest/_modules/monai/apps/deepgrow/transforms.html#AddInitialSeedPointd) -> This transform adds random guidance as initial seed point for a given label,
+  > - [AddGuidanceSignald](https://docs.monai.io/en/latest/_modules/monai/apps/deepgrow/transforms.html#AddGuidanceSignald) -> This transform adds guidance signal (foreground and background clicks) for input image, 
+  > - DiscardAddGuidanced -> This transform discards foreground and background tensors with a probability define as an argument.
```
Comment (diazandr3s) [inline sample-apps/generic_deepgrow/README.md:45]: I think this is particular to the DeepEdit App.

File: sample-apps/generic_deepgrow/README.md (hunk 2 above)
Comment (finalelement) [reply sample-apps/generic_deepgrow/README.md]: Removed

File: sample-apps/generic_deepgrow/README.md (hunk 3 above)
Comment (finalelement) [reply sample-apps/generic_deepgrow/README.md:47]: Except for the last one, I think. Removed DiscardAddGuidanced

**Pull request merged**